### ISSUE TITLE
find block variable declaration

### ISSUE THREAD

**Issue by rxie25** [rxie25/PLT2016Spring#41]
currently variable declaration in find block will not generate double in front of it.
```c
SomeContext = {
  a = { 43; }
}

SomeContext:find a {
  b = 42;
  c = 41;
  d = 40;
  printf("%.0f\n", a);
  printf("%.0f\n", b);
  printf("%.0f\n", c);
  printf("%.0f\n", d);
}
```
will generate
```c
#include <stdio.h>
#include <math.h>
void find_SomeContext_0 () {
  double a;

a = 43.;

b = 42.;
c = 41.;
d = 40.;
printf("%.0f\n", a);
printf("%.0f\n", b);
printf("%.0f\n", c);
printf("%.0f\n", d);
}

int main() {
find_SomeContext_0 ();
 return 0;
}
```

**Comment by rxie25** [comment]
#72

**Comment by rxie25** [comment]
It will be resolved #56
